politeiawww/api/pi/v1/v1.go

```diff
@@ -72,3 +155,67 @@ type VoteMetadata struct {
 	// in the runoff vote.
 	LinkTo string `json:"linkto,omitempty"`
 }
+
+var (
+	// BillingStatuses contains the human readable billing statuses.
+	BillingStatuses = map[BillingStatusT]string{
```

Comment:
Prefer this goes underneath the billing status declarations.  Declare the billing statuses first then declare their human readable equivalents.